zk_prover/src/circuits/traits.rs

```diff
@@ -30,4 +30,24 @@ pub trait CircuitBase {
             |mut region| region.assign_advice(|| "value", advice_col, 0, || Value::known(value)),
         )
     }
+
+    /// Loads the lookup table with values from `0` to `2^8 - 1`
+    fn load(&self, layouter: &mut impl Layouter<Fp>, column: Column<Fixed>) -> Result<(), Error> {
```

Comment:
Technically this is not a problem, but this method is very specific to the implementation of our MST. It makes `CircuitBase` not a common trait anymore but a specific API for this case, so either naming and description don't correspond to the function anymore, or the method should be inside the MST circuit implementation.

Reply:
I guess the trait is a leftover from the time when we had two circuits